### PR TITLE
Add a widget

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
@DevExpress/testcafe 

Test at http://strelyaev-w8:8080/testcafe/documentation/using-testcafe/programming-interface/

### CODE REVIEW

File: src/scripts/widget.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    function sendGaEvent(event) {
+        {% if jekyll.environment == "production" %}
+
+        ga('send', 'event', 'widget', event);
+
+        {% endif %}
+    }
+
+    function updateWidgetVisibility() {
+        if(window.needShowShareWidget()) {
```
Comment (AlexanderMoskovkin) [inline src/scripts/widget.js]: `if(window.localStorage.getItem(WIDGET_DISPLAYED_FLAG) || window.needShowShareWidget())`

Comment (VasilyStrelyaev) [inline src/scripts/widget.js]: don't understand why

Comment (AlexanderMoskovkin) [inline src/scripts/widget.js]: If you already shown the widget there is no need to check `needShowShareWidget`. Just show it again

Comment (VasilyStrelyaev) [inline src/scripts/widget.js]: We need to. For example, we need to check if we are in the documentation and not in Getting Started

**Pull request merged**